src/component.d.ts

```diff
@@ -74,31 +82,25 @@ declare namespace Component {
 
   interface Input {
     [key: string]: (this: ComponentInstance, event: KeyboardEvent) => void,
-    /**
-     * Catch all input function
-     *
-     * Will be invoked when there is no dedicated function for a certain key
-     */
-    'any'?: (this: ComponentInstance, event: KeyboardEvent) => void,
```

Comment:
I think this one shouldn't be removed. This is not a TS `any`, but there is an actual key method called `any` that acts as a catch all key handler ..

Reply:
Oh, you are right. I will get it back.